Makefile

```diff
@@ -611,3 +614,13 @@ buildsnapcraftready:
 	cd _build/dotsnap$(PREBUILT_DIR)/usr/bin; ln -s ../..$(POPLOG_VERSION_DIR)/pop/pop/poplog .
 	cp snapcraft.yaml _build/dotsnap	
 
+
+################################################################################
+# Perform a GitHub release via CircleCI (you need to be authorised to use this)
```

Comment:
> Perform a GitHub release via CircleCI (you need to be authorised to use this

->

> Perform a GitHub release by tagging a commit which triggers CircleCI to build and push binaries for the tag's GitHub release

I'm not sure what the authorized bit is meant to mean? Maybe say "you must be authorized to push tags to the upstream repository on GitHub to perform this action"

Reply:
Done.